src/Profiles.ts

```diff
@@ -45,13 +43,15 @@ export class Profiles { // Processing stops if there are no profiles detected
         return Profiles.loader;
     }
     private static loader: Profiles;
+    private static spawnValue: number = -1;
     public allProfiles: IProfileLoaded[] = [];
     public defaultProfile: IProfileLoaded;
 
-    private spawnValue: number = -1;
-    private initValue: number = -1;
+    private initValue: number = 0;
+    private profileManager;
```

Comment:
Should we think about exposing the profileManager from this class since it's where it's initialized?

Reply:
I don't understand why we would want it accessed from outside of this instance so I was locking it down.